src/commentbot/CommentBot.jl

```diff
@@ -94,10 +94,25 @@ function make_pull_request(pp::ProcessedParams, rp::RequestParams, rbrn::RegBran
     repo = join(split(target_registry["repo"], "/")[end-1:end], "/")
     pr, msg = create_or_find_pull_request(repo, params, rbrn)
     tag = tag_name(ver, subdir)
-
+    
+    releasenotes_note if isempty(rp.release_notes)
+        """
+        
+        ### Tip: Release Notes
+        
+        Did you know you can add release notes too? Just add markdown formatted text underneath the comment and it will be added to 
```

Comment:
I think this isn't enough; you need to format it to pass this regex: https://github.com/JuliaRegistries/Registrator.jl/blob/1558fb7a627a85eee6b58950e507daf54efb9dfe/src/commentbot/param_types.jl#L49

like described here: https://github.com/JuliaRegistries/Registrator.jl#release-notes.

But maybe we could relax that regex, or just mention they should write `"Release notes:"`

Reply:
Ah ok. I updated it. If JuliaRegistrator writes `@JuliaRe...` as an example, will that cause the universe to collapse?

Reply:
If you put the ping inside of backticks, JuliaRegistrator shouldn't actually get pinged, so you should be good there.

Reply:
But if the ping isn't in backticks, I actually do think that the bot will get caught in a loop replying to itself 😂 

And eventually hit its rate limit, causing all registrations to grind to a halt.

Reply:
uhhh maybe 🤔. I think in https://github.com/JuliaRegistries/Registrator.jl/blob/1558fb7a627a85eee6b58950e507daf54efb9dfe/src/commentbot/CommentBot.jl#L188 we could bail if the user is JuliaRegistrator. Or maybe here: https://github.com/JuliaRegistries/Registrator.jl/blob/1558fb7a627a85eee6b58950e507daf54efb9dfe/src/commentbot/param_types.jl#L20. Not totally sure though

Reply:
Yeah I was just thinking that. I'll try to make a PR.

Reply:
https://github.com/JuliaRegistries/Registrator.jl/pull/361